sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/models/TableStorageException.java

```diff
@@ -5,7 +5,8 @@
 
 import com.azure.core.exception.HttpResponseException;
 import com.azure.core.http.HttpResponse;
-import static com.azure.storage.common.implementation.Constants.HeaderConstants.ERROR_CODE;
+
+import static com.azure.data.tables.implementation.StorageConstants.HeaderConstants.ERROR_CODE;
 
 /**
  * A {@code TableStorageException} is thrown whenever the Tables service successfully returns an error code that is not
```

Comment:
Isn't `TableStorageException` deleted now?

Reply:
That work is part of a different PR.

Reply:
#21339